temporalio/testing/_activity.py

```diff
@@ -74,15 +74,30 @@ def __init__(self) -> None:
         self._cancelled = False
         self._worker_shutdown = False
         self._activities: Set[_Activity] = set()
+        self.cancellation_details: (
+            temporalio.activity._ActivityCancellationDetailsHolder
+        )
```

Comment:
```suggestion
        self._cancellation_details = temporalio.activity._ActivityCancellationDetailsHolder()
```

Don't want to expose to users, and we want to go ahead and instantiate the holder